.github/workflows/ci-module.yml

```diff
@@ -14,13 +14,13 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu, windows, macos]
-        node: ${{ inputs.min-node-version == 14 && fromJSON('["*", "16", "14"]') || fromJSON('["16", "14", "12"]') }}
+        node: ${{ inputs.min-node-version == 14 && fromJSON('["*", "20", "18", "16", "14"]') || fromJSON('["16", "14", "12"]') }}
```

Comment:
With the latest "setup-node", it [supports `"latest"` to get the latest node release](https://github.com/actions/setup-node#supported-version-syntax), rather than the latest locally installed which `"*"` seems to point to. Ie. right now it would resolve to v21.1.0, vs. 20.9.0 that `"*"` resolves to.

If you change this, remember to update the `check-latest: ${{ matrix.node == '*' }}` line.

Reply:
The use of `*` was to get a behaviour close to what the new `latest` keyword support. We should probably migrate to it then IMO. Thanks for mentioning it Gil, I missed it when going through their changelog.

Reply:
Is this all right with the last change?